DigitalLearningSolutions.Web/appsettings.Development.json

```diff
@@ -1,6 +1,6 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "Data Source=localhost;Initial Catalog=mbdbx101;Integrated Security=True;"
+    "DefaultConnection": "Data Source=localhost;Initial Catalog=mbdbx101_test;Integrated Security=True;"

```

Comment:
Hmm.. Is this intended?

Reply:
Nope was checking something, will revert